controllers/clusterversion_controller.go

```diff
@@ -604,6 +624,63 @@ func labelClientOperatorSubscription(c *ClusterVersionReconciler) error {
 		}
 	}
 
+	c.subscriptionChannel = sub.Spec.Channel
```

Comment:
This function is supposed to have no side-effect as it is a pure add-label operation. It is a bad add to have invisible side-effects to an establish flow

Reply:
agreed! I was also doubtful.

Reply:
done.